src/router.ts

```diff
@@ -622,10 +623,9 @@ const routeVoterToSlackChannel = async (
     userInfo[userInfo.activeChannelId],
     userInfo.activeChannelId
   );
-  await DbApiUtil.setThreadNeedsAttentionToDb(
+  await DbApiUtil.setThreadRoutedToDb(
```

Comment:
can you move both this and getThreadNeedsAttentionFor to the very end of this function so it gets to the production-critical stuff first?

Reply:
We can't move the getThreadNeedsAttentionFor very far because we need the value to set up the new thread.  I can move the setThreadRouted call by getting rid of the early return.

I'm not sure it's worth thinking too hard about what happens if this is interrupted when it pretty much *all* of the code assumes things aren't interrupted.